sympy/stats/stochastic_process_types.py

```diff
@@ -84,6 +88,18 @@ def _sym_sympify(arg):
     else:
         return _sympify(arg)
 
+def random_indexed_symbols(expr):
+    """
+    Returns all RandomIndexedSymbols within a SymPy Expression.
+    """
+    atoms = getattr(expr, 'atoms', None)
```

Comment:
This function returns random indexed symbols from the expression, So I thought using this might be cleaner than using `expr.atoms(...)`. I will remove it and use `expr.atoms(..)`

Reply:
For these things it's usually better to just use `sympy/core` functions. Using `sympy/core` makes the code easier to read, in my opinion.

But we can also keep this function if you are calling it many times. It's just a code style preference...

Reply:
(but definitely don't use `getattr`... it makes code analysis more complicated)